var/spack/repos/builtin/packages/cosma/package.py

```diff
@@ -12,32 +12,30 @@ class Cosma(CMakePackage):
     Distributed Communication-Optimal Matrix-Matrix Multiplication Library
     """
 
-    maintainers = ['teonnik', 'kabicm']
+    maintainers = ['haampie', 'kabicm', 'teonnik']
     homepage = 'https://github.com/eth-cscs/COSMA'
-    url = 'https://github.com/eth-cscs/COSMA/releases/download/v2.0.2/cosma.tar.gz'
+    url = 'https://github.com/eth-cscs/COSMA/releases/download/v2.2.0/cosma.tar.gz'
     git = 'https://github.com/eth-cscs/COSMA.git'
 
     # note: The default archives produced with github do not have the archives
     #       of the submodules.
     version('master', branch='master', submodules=True)
+    version('2.2.0', sha256='1eb92a98110df595070a12193b9221eecf9d103ced8836c960f6c79a2bd553ca')
+    version('2.0.7', sha256='8d70bfcbda6239b6a8fbeaca138790bbe58c0c3aa576879480d2632d4936cf7e')
     version('2.0.2', sha256='4f3354828bc718f3eef2f0098c3bdca3499297497a220da32db1acd57920c68d')
-    # note: this version fails to build at the moment
-    # version('1.0.0',
-    #         url='https://github.com/eth-cscs/COSMA/releases/download/1.0/cosma.tar.gz',
-    #         sha256='c142104258dcca4c17fa7faffc2990a08d2777235c7980006e93c5dca51061f6')
 
-    variant('cuda', default=False,
-            description='Build with the CUBLAS back end.')
+    variant('gpu', default=False,
```

Comment:
Why change the name from CUDA to GPU? Most Spack packages use the `cuda` variant. In fact, there is a `CudaPackage` base class you can extend that automatically adds a `cuda` variant.

Reply:
The reason we changed to `gpu` is because our library also supports AMD GPUs through `rocblas`. So, when `rocblas` becomes available in spack, we would add it as an option. But until then, I can change it back to `cuda` if you prefer.

Reply:
In that case, it's up to you and the other maintainers. If you want a single `gpu` variant than controls both cuda and rocblas, that's fine. If you want separate variants for cuda and rocblas that's also fine. Thanks for clarifying!

Reply:
Thanks @adamjstewart. It makes sense to have separate variants for those as you proposed, so I renamed the variant back to `cuda`.